### DISCUSSION TITLE
Error 400 Param "code" is not optional.  Error ID: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

### DISCUSSION THREAD

**Discussion by akanksha-sharma-main** [appwrite/appwrite#5817] [Q&A]
Error 400
Param "code" is not optional.

Error ID: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
when oauth from github

**Comment by stnguyen90** [comment]
@akanksha-sharma-main, this usually happens because something is misconfigured. The URL might give you some clues as to what's wrong.

If you're still having trouble, I recommend joining our [Discord](https://appwrite.io/discord) server and creating support post.
